src/backend/distributed/shared_library_init.c

```diff
@@ -448,16 +448,18 @@ RegisterCitusConfigVariables(void)
 		NULL, NULL, NULL);
 
 	DefineCustomEnumVariable(
-		"citus.copy_transaction_manager",
-		gettext_noop("Sets the transaction manager for COPY into distributed tables."),
-		gettext_noop("When a failure occurs during when copying into a distributed "
-					 "table, 2PC is required to ensure data is never lost. Change "
-					 "this setting to '2pc' from its default '1pc' to enable 2PC."
-					 "You must also set max_prepared_transactions on the worker "
-					 "nodes. Recovery from failed 2PCs is currently manual."),
-		&CopyTransactionManager,
-		TRANSACTION_MANAGER_1PC,
-		transaction_manager_options,
+		"citus.multi_shard_commit_protocol",
+		gettext_noop("Sets the commit protocol for commands modifying multiple shards."),
+		gettext_noop("When a failure occurs during commands that modifies multiple "
+					 "shards (currently, only COPY on distributed tables modify more "
```

Comment:
Github line comments are a little confusing. The comment is usually about the last line that you see above.